submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/DynamicExtSubmissionSubmitter.kt

```diff
@@ -0,0 +1,46 @@
+package ac.uk.ebi.biostd.submission.domain.service
+
+import ac.uk.ebi.biostd.common.properties.TaskHostProperties
+import ac.uk.ebi.biostd.submission.domain.extended.ExtSubmissionSubmitter
+import ac.uk.ebi.biostd.submission.domain.extended.LocalExtSubmissionSubmitter
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mu.KotlinLogging
+import java.io.File
+import java.util.UUID
+
+private val logger = KotlinLogging.logger {}
+
+class DynamicExtSubmissionSubmitter(
+    private val extSubmissionSubmitter: LocalExtSubmissionSubmitter,
+    private val hostProperties: TaskHostProperties,
+) : ExtSubmissionSubmitter by extSubmissionSubmitter {
+
+    override suspend fun processRequest(accNo: String, version: Int) {
+        when (hostProperties.enableTaskMode) {
+            true -> copyRequestFilesRemotly(accNo, version)
```

Comment:
I think the dynamic submitter should be able to execute any stage, not only copying the files. Or at least that's what I understood from the previous code.

Reply:
given the fact that executing other steps has other implications I do not see the point make implementation more complex by increaasing the scope of the feature  to "be able to execute any stage in the data mover". Right now is only the persitence